### ISSUE TITLE
New TURBO RANKED MODE!

### ISSUE THREAD

**Issue by andrestrindade** [ValveSoftware/Dota2-Gameplay#8751]
I am writing to suggest a new addition to the game that I believe would greatly enhance the player experience: a Ranked Turbo Mode.

Turbo Mode has become increasingly popular among Dota 2 players as a way to play faster, more action-packed games. However, the lack of ranking system in Turbo Mode makes it less appealing to competitive players who enjoy the thrill of climbing the ranks.

By introducing a Ranked Turbo Mode, players can enjoy the fast-paced gameplay of Turbo Mode while also being able to compete for higher ranks and rewards. This would attract a new wave of competitive players who may have previously overlooked Turbo Mode.

Additionally, the shorter game time in Turbo Mode makes it more accessible to players who may not have hours to spend on each game. The Ranked Turbo Mode would be perfect for those players who want to enjoy a quick and intense game while also having a chance to climb the ranks.

I believe that the addition of a Ranked Turbo Mode would greatly enhance the Dota 2 experience for both casual and competitive players alike. It would provide a new and exciting way to play the game, and attract a wider range of players to the community.

Thank you for considering this suggestion, and I look forward to seeing what the future holds for Dota 2.

**Comment by zarqk** [comment]
+, good idea

**Comment by jonnik666** [comment]
I support this under one condition: W = +1 MMR; L = -15 MMR

**Comment by undestructable100** [comment]
SAPPORT

**Comment by lokize** [comment]
+1 for this

**Comment by megasterno** [comment]
It would be great if this is implemented in a way that Turbo has its MMR that does not affect Normal Ranked Mode MMR. 
So, any player that plays Turbo (like me) can be paired with other Turbo players that might not play Normal Ranked and have no MMR but certainly have some **Turbo skills** like:
- Selling wards (instead of planting) to confuse the mid-laner.
- Sending the courier to the side shop to get a dropped tru-sight gem faster.
And other skills that are only used/acquired while playing Turbo.

**Comment by Vittor-Javidan** [comment]
2024, and I still hope for turbo ranked Valve!

**Comment by KAMRONBEK** [comment]
2024, I really like this idea! Adding a _Ranked Turbo_ Mode to Dota 2 would be amazing for both serious and casual players. It mixes the quick fun of Turbo Mode with the challenge of ranked games. This would be great for players who don’t have a lot of time but still want to play seriously. It could also bring back players who left because normal games take too long. I think this new mode would make the game more exciting and keep it interesting. I hope the developers think about adding this. Great suggestion!

**Comment by Vittor-Javidan** [comment]
They should really add this new ranked mode. I mean, they could just add a mmr system for turbo mode and it's done. No need to change anything about the current ranked and casual modes.

Seriously, this should be the easiest implementation of valves life. This will benefit ppl who just want to play fast games, and those who don't have a large amount o time to play due adult life reason.

I bet lot of players do not play ranked games anymore, because we feel the risk of throwing 40 min of our life on garbage if there is a grieffer. On turbo, if there is one grieffer, you only need to pass like 10-20 min max to start another game.

**Comment by begyy** [comment]
we support this

**Comment by 498rahim** [comment]
We support this, good idea

**Comment by Vittor-Javidan** [comment]
We support this 3

**Comment by andrestrindade** [comment]
It can be in a different ranking system; While we have a rank for regular matches, we can have a rank for Turbo matches.

**Comment by benja11y** [comment]
100% support this @gdrewb-valve

**Comment by lokize** [comment]
any news guys?

**Comment by Vittor-Javidan** [comment]
> any news guys?

nope. But maybe if this post became popular, maybe they can implement this, since it must be a easy implementation to the game, with huge potential of adoption. We had 2 mmr system in the past, party and solo. They could just do the same, but for normal and turbo games.

**Comment by Vittor-Javidan** [comment]
I created a new discussion with the tag "idea". Upvote this new one, so the idea goes up guys. I feel like a issue became lost in many other issues, since it is sorted by date.

https://github.com/ValveSoftware/Dota2-Gameplay/discussions/21650

**Comment by MMovich** [comment]
This is stupidity. There will be the same problems as with All Pick rating games (fighting for a role, for example)

**Comment by Ludovick127** [comment]
It's done here
www.dotaturborank.com

Rust powered 👍

**Comment by Novaraaa** [comment]
Please consider adding a separate Ranked Turbo queue.

Turbo is essentially the only mode I regularly want to play, but I still want the competitive progression, matchmaking and sense of achievement that comes with Ranked.

For players who love Dota but don't always have time for 40–60+ minute matches.

I would genuinely play Ranked Turbo as my main competitive mode if you guys at Valve added it.

Even testing it as a seasonal or experimental queue would be worth trying.

**Comment by Ludovick127** [comment]
This is why I have built an independant Turbo Leaderboard here: www.dotaturborank.com
Check on Google

**Comment by Novaraaa** [comment]
Thank you. I should have read the whole thread. 

This is really nice and a step in the right direction :) I'll be using it.

**Comment by Ludovick127** [comment]
Tonbe transparent this is not an official ranking, and it has no impact on Dota 2 matchmaking.
The platform tracks Turbo matches played around the world (more than 1000/min) and calculates an independent ELO rating for each player. All ranking rules and calculations are explained on the website, with an Android app coming soon.
The global leaderboard is updated every 10 minutes and resets at the beginning of each month. At the end of every season, the top 3 players from each region are crowned regional winners.
There is also a permanent Hall of Fame that keeps track of the best players across previous seasons.
In addition, the platform provides dynamic Turbo hero builds based on data from some of the best Turbo players in the world, helping players discover the most effective item, skill, and talent choices.
More features are coming soon.

**Comment by Novaraaa** [comment]
Thank you for all the extra info. 

I looked into it and saw how it basically works and it is a really cool idea and indeed scratches the ranked itch a bit. 

Will keep following. Appreciated it once again, @Ludovick127.
